src/pages/understand-stacks/stacking-using-CLI.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Stacking using the Stacks CLI
+description: Learn how use the Stacks CLI to participate in stacking
+experience: beginner
+duration: 10 minutes
+tags:
+  - guide
+images:
+  sm: /images/pages/stacking-rounded.svg
+---
+
+## Introduction
+
+!> The Stacking implementation is still in development and could change in the coming weeks
+
+In this guide, you'll learn how to use the Stacks CLI to participate in stacking. The CLI is a great way to quickly try out stacking on testnet. To integrate stacking into your application, see the [stacking integration guide](/stacks-blockchain/integrate-stacking).
+
+## Requirements
+
+First, you'll need to understand the [Stacking mechanism](/stacks-blockchain/stacking).
+
+You'll also need [NodeJS](https://nodejs.org/en/download/) `12.10.0` or higher to complete this tutorial. You can verify your installation by opening up your terminal and run the following command:
+
+```bash
+$ node --version
+```
+
+You will also need to install the Stacks CLI from NPM:
+
+```bash
+$ npm install @stacks/cli -g
+```
+
+## Generate A New Key Pair
+
+```bash
+$ stx make_keychain -t
+{
+  "mnemonic": "turn food juice small swing junior trip crouch slot wood nephew own tourist hazard tomato follow trust just project traffic spirit oil diary blue",
+  "keyInfo": {
+    "privateKey": "dca82f838f6e5a893cffc8efe861196252373a5b8b62c0b55ba3a0a7a28795d301",
+    "address": "ST1P3HXR80TKT48TKM2VTKCDBS4ET9396W0W2S3K8",
+    "btcAddress": "mqPBWmSGJhrA9x5XJC6qJtFsHnudqZ2XJU",
+    "index": 0
+  }
+}
+```
+
+We'll be using this testnet key pair to perform stacking. But first we'll need to get some testnet tokens.
+
+-> If you're stacking on mainnet, make sure you have an account with sufficient number of Stacks tokens to participate.
+
+## Get Testnet Tokens Using The Faucet
+
+Use the following `curl` command to request tokens from the testnet node's faucet endpoint.
+
+Note that we used the address we generated above as a parameter.
```

Comment:
I'd use the visual note component for that. Start the line with `-> Note that ...`